content/explanations/gamification/ranks.md

```diff
@@ -0,0 +1,68 @@
+---
+kind: explanation
+sidebar: ranks-honor
+prev: /explanations/gamification/
+next: /explanations/gamification/honor/
+---
+
+# Ranks
+
+Ranks are used to indicate the proficiency of users and the difficulty of Kata. There are two classes of ranks, Kyu and Dan, which are divided in 8 levels each. By increasing order of proficiency/difficulty:
+
+- 8 Kyu to 1 Kyu
+- 1 Dan to 8 Dan
+
+Why the names Kyu and Dan? The terms are borrowed from a system in Japanese martial arts, which is in turn borrowed from the game of Go. Kyu (or Kyū) indicates the number of degrees away from master level (Dan). This is why they count downward. Once you reach master level, we count upward. Black belts in martial arts are Dan level.
+
+## User Rank Breakdown
+
+When you visit your profile on Codewars, you can see that you have an Overall rank as well as individual ranks for each language you have completed kata in:
+
+<div class="block dark:hidden">
+
+![rank progress](./img/rank-breakdown_light.png)
+
+</div>
+<div class="hidden dark:block">
+
+![rank progress](./img/rank-breakdown_dark.png)
+
+</div>
+
+The wheel on the left indicates your progress toward your next rank. For example, if you see the `1 dan` bagde in the wheel and your overall rank is `1 kyu / 70.0%` that means you have earned 70% of the progress needed to reach the rank of 1 dan.
+
+The overall rank increases each time you successfully complete kata you never completed in any language before. This increase occurs only one time per kata. On the other hand, the language ranks are language specific, so if you complete the same Kata in several languages, each one of them will progress accordingly.
+
+Note that you cannot gain progress if you forfeited a kata.
+
+## Leveling Your Rank
+
+The score used to determine your rank is not the same as honor. You can't see your score on your profile, but it is visible in the API at the endpoint `https://www.codewars.com/api/v1/users/USERNAME`.
+
+As said before, the only way to "rank up" is to complete Kata. However, the rank of the Kata you complete has a huge difference on the evolution of your rank/%.
```

Comment:
`%` as a word does not look good IMO, i'd go with "percentage" or "rank progress \[percentage\]", or something like that.